src/stacks-hierarchy/Root_ChooseTicketReceiverScreen/Root_ChooseTicketReceiverScreen.tsx

```diff
@@ -58,12 +74,12 @@ export const Root_ChooseTicketReceiverScreen: React.FC<Props> = ({
           <PhoneInput
             submitButtonText={t(PurchaseOverviewTexts.summary.button)}
             submitButtonTestId="toPaymentButton"
-            // TODO: this is just a placeholder, update when API is ready
-            onSubmitPromise={() => new Promise((r) => setTimeout(r, 1000))}
-            onSubmitAction={(number) => {
+            onSubmitPromise={(number) => handleValidatePhoneNumber(number)}
```

Comment:
As far I understand, this is a validation of the phone, then if ok call the action. In that case I would suggest better names for it. ValidatePhoneAction, OnCompletedPhoneValidationAction or something like that... or

Reply:
Updated, thanks!